lib/main.js

```diff
@@ -13,11 +13,12 @@ function _createWindow (options) {
     show: false
   }, options)
 
-  opts.preload = path.join(__dirname, 'renderer-preload')
   if (v[0] === 0 && v[1] >= 37) {
     opts.webPreferences = assign({
       preload: path.join(__dirname, 'renderer-preload')
     }, options.webPreferences)
+  } else {
```

Comment:
Something like this might work better:
``` javascript
if (v[0] === 0 && v[1] < 37) {
  opts.preload = path.join(__dirname, 'renderer-preload')
} else {
  opts.webPreferences = assign({
    preload: path.join(__dirname, 'renderer-preload')
  }, options.webPreferences)
}
```